flapp-webapp/src/components/CaseMgmt.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div class="fill-height-lg survey-container contentcontainer codecontainer">
+    <h1>Case Management form</h1>
+    <p>{{ message }}</p>
+    <p>{{ message2 }}</p>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "CaseMgmt",
+  data: function() {
+    return {
+      message: "This is a case management form",
+      message2: "coming soon"
+    };
+  }
+};
+</script>
+
+<!-- Add "scoped" attribute to limit CSS to this component only -->
+<style scoped>
```

Comment:
This block is repeated, it would be better to move it to styles.scss if it's an application global style. There are also bootstrap classes for adjusting margins, this this case something like mt-4 (https://getbootstrap.com/docs/4.0/utilities/spacing/)